### DISCUSSION TITLE
Welcome to FlashBrowser2 Discussions!

### DISCUSSION THREAD

**Discussion by radubirsan** [radubirsan/FlashBrowser#1] [Announcements]
## 👋 Welcome!
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jidanni** [comment]
https://flash.pm/2021/09/23/run-flash-player-on-linux-ubuntu-with-flashbrowser-in-14-steps/ says
> Click the download button it will take you to the GitHub repo

No, doing that will simply download the .exe.

**Reply by radubirsan** [reply]
Thank you I will have to make a new tutorial with the new website

**Comment by Regis-Steigleder** [comment]
Hello everyone.
My name is Paulo Regis. Im from Brazil. I develop applications using flash. I was surprised last Monday (03/17/25) , Firefox Portable blocked Flash Player sudenly.
I found this great browser that works perfectly.
I have a question about displaying PDF files. Is it possible to display PDF files in this browser? How can I do it.
